aws/resource_aws_emr_cluster.go

```diff
@@ -486,6 +502,7 @@ func resourceAwsEMRClusterRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	d.Set("name", cluster.Name)
+
```

Comment:
We do still want the `d.Set("scale_down_behavior", cluster.ScaleDownBehavior)` here so Terraform is always updating its state correctly from the API each read as well as on import. We just didn't need the wrapping `if d.Get()` check that was previously in the PR. Sorry I wasn't more clear about what I meant earlier! 😄